### DISCUSSION TITLE
CUE Team at KubeCon EU in Paris, March 20-22 2024

### DISCUSSION THREAD

**Discussion by mvdan** [cue-lang/cue#2968] [Announcements]
@myitcv and @mvdan will be in Paris from Tuesday evening until Friday for [KubeCon EU](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/).

If you plan to be there as an attendee or are otherwise in the area and would like to meet to talk about CUE, please comment in the thread, ping either of us on Slack, or email mvdan@cue.works.

**Comment by myitcv** [comment]
Want to talk modules? Want to see a demo of the Central Registry? 

Drop by and speak with @mvdan and @myitcv at 1130 UTC+1 (Paris time).

We will again be in the "quiet" part of the Solutions Showcase on level 7.2, in "Arc de Triomphe" area, next to the SpiceDB / authzed stand, with the green high chairs.

![PXL_20240320_104207092](https://github.com/cue-lang/cue/assets/3374574/afac8b7c-2825-49aa-88e9-6cc5635f75b3)
![PXL_20240320_104604871](https://github.com/cue-lang/cue/assets/3374574/08c93e32-8124-4bd7-a917-23466c55687d)
![PXL_20240320_104825331](https://github.com/cue-lang/cue/assets/3374574/f51f2a92-e198-44ac-adba-4ef07e77cb4f)

**Reply by myitcv** [reply]
Our green chairs have been replaced by a massage station! 

So we are now on the benches located behind the white tents:

![PXL_20240321_103432526](https://github.com/cue-lang/cue/assets/3374574/6f1d3c55-a243-458b-8cb6-86cc350b93b6)

**Reply by mvdan** [reply]
Evidence courtesy of @florianl!
![PXL_20240321_103603414](https://github.com/cue-lang/cue/assets/3576549/38395dd2-4721-4bf5-9c15-28c7e6667a79)

**Comment by mvdan** [comment]
We have set up camp in the same location this morning, come say hi! Message either @myitcv or myself if you can't find us :)

**Reply by hansbogert** [reply]
Ah man, why am I seeing this only now? Would've loved to talk to you guys. What's your guys take on this past Kubecon? I kinda missed talks on technology like Cue( of course Stefan did a nice one though!). The talks on AI and scheduling problems was way over represented imo. And also, did you get any interesting questions/discussions?

**Reply by myitcv** [reply]
Sorry to have missed you, @hansbogert! I've not been to many KubeCons, but this one was quite different and I can't fully explain why. One very noticeable thing is that the solutions showcase was considerably louder than Chicago last year, and indeed Amsterdam the year before. I think that made it much harder to have idle conversations at a booth, because it was so much more a scrum and struggle to be heard! We would love to do some specific CUE talks and work out the best format at the most appropriate conference. Anything in particular you had in mind?

**Reply by hansbogert** [reply]
Personally I hardly every go to conferences tbh and I just watch the talks on (probably) Youtube. A specific conference does not come to mind.

The format which I think would be good for Cue in its current stage would be to continue what already has been done in the documentation recently; short but independent tutorials which really show the power of Cue.

Coming from a  CS and software developer background and  now SRE, what strikes me is that self-taught professionals in a SRE and devops role dismiss the safety advantage that typing gives. A talk which shows everyday pain-points which can be remedied by Cue and its strict typing could be advantageous.

I think Roger Peppe's talk [[1]](https://www.youtube.com/watch?v=I5cEXqa4ZOI) is a good example, though this was somewhat geared towards application developers. A similar talk for SRE/Devops would work wonders I think.  

Thank you for Cue!
